src/models/user-game-list-response.model.ts

```diff
@@ -0,0 +1,26 @@
+
+export interface GameListResponseGame {
+  __typename: "GameLibraryTitle";
+  conceptId: string;
+  entitlementId: string|null;
+  image: {
+    __typename: "Media";
+    url: string
+  };
+  isActive: boolean|null;
+  lastPlayedDateTime: string;
```

Comment:
We should document this like other date time values so it shows up properly when using IntelliSense in a dev's editor.

Check _trophy-group-earnings.model.ts_ for a good example.